db/migrate/20150528113122_add_image_relation_to_container.rb

```diff
@@ -0,0 +1,9 @@
+class AddImageRelationToContainer < ActiveRecord::Migration
+  def up
+    add_column :containers, :container_image_id, :bigint
```

Comment:
Please consider to remove the column `image` that now is holding a duplicated information (the image name).